Dockerfile

```diff
@@ -319,6 +347,7 @@ RUN oc completion bash > /etc/bash_completion.d/oc
 RUN ocm completion > /etc/bash_completion.d/ocm
 RUN osdctl completion bash --skip-version-check > /etc/bash_completion.d/osdctl
 RUN yq --version
+RUN k9s version
```

Comment:
If k9s has command line completion we should add that as well

Reply:
fixed in [add k9s bash completion](https://github.com/openshift/ocm-container/pull/189/commits/1449bad9702ec36cc9fa138e96e38da5d9fadd69) 
since it's a TUI, completion doesn't offer much:
```
$ k9s 
completion  (Generate the autocompletion script for the specified shell)  info        (Print configuration info)
help        (Help about any command)                                      version     (Print version/build info)

```